src/index.ts

```diff
@@ -10,6 +10,7 @@ export interface Config<P extends object, A extends object> {
   comp?: React.ComponentType<any>
   child?: React.ComponentType<any>
   childProps?: ((props: any) => any) | any
+  omitProps?: string[]
```

Comment:
would it be possible to define omitProps as `keyof P[]` or are these potentially props that cannot be statically known?